app/utils/fields-validations.ts

```diff
@@ -0,0 +1,22 @@
+import Joi from '@hapi/joi';
+import i18n from '../../locale/en.json';
+
+function homeOfficeNumberValidation(reference: string) {
+  const schema = Joi
+    .string()
+    .required()
+    .regex(/^[a-zA-Z]{2}[0-9]{6}$/)
```

Comment:
Not sure if this is the correct expression:
this will allow Ids like `AA1234567`
Reading the description of the ticket it says the ids should be the following:
`A123457` - but it might be outdated so it'll be worth double-checking this.
regex suggestion for the above.
```suggestion
    .regex(/^[a-zA-Z][0-9]{7}$/)
```